spec/models/authenticator/httpd_spec.rb

```diff
@@ -403,6 +403,12 @@ def authenticate
           expect(-> { authenticate }).to change { User.where(:userid => 'bob@example.com').count }.from(0).to(1)
         end
 
+        it "logs the success" do
+          allow($log).to receive(:info).with(/Event/)
+          expect($log).to receive(:info).with(/User creation successful for User$/)
```

Comment:
Should this expectation be on `$audit_log`?

Reply:
Also, the `$` at the end of the regex shouldn't be there since that won't be the end of the message, right?

Reply:
Also also, should we expect that there is an event queued?

Reply:
We get the following in `evm.log`

```
[----] I, [2018-10-16T15:16:17.864540 #10740:3fc38f840e30]  INFO -- : MIQ(MiqQueue.put) Message id: [327],  id: [], Zone: [Zone 1], Role: [], Server: [], MiqTask id: [], Ident: [generic], Target id: [], Instance id: [], Task id: [], Command: [MiqEvent.raise_evm_event], Timeout: [600], Priority: [100], State: [ready], Deliver On: [], Data: [], Args: [["MiqServer", 315], "user_created", {:event_details=>"User creation successful for User: Bob Builderson with ID: bob@example.com"}]
```

note `"user_created", {:event_details=>"User creation successful for User: Bob Builderson with ID: bob@example.com"}]`

Yes I should be expecting an event queue too, I'm not sure how to do that.